datafeeder/datafeeder.properties

```diff
@@ -0,0 +1,143 @@
+#datafeeder.publishing.geonetwork.template-record-id:
+datafeeder.publishing.geonetwork.template-record: file:${georchestra.datadir}/datafeeder/metadata_template.xml
+datafeeder.publishing.geonetwork.template-transform: file:${georchestra.datadir}/datafeeder/metadata_transform.xsl
+
+
+# GeoTools DataStore connection params used inside datafeeder when importing
+# uploaded datasets to the target store
+datafeeder.publishing.backend.local.dbtype=postgis
+datafeeder.publishing.backend.local.host=${pgsqlHost}
+datafeeder.publishing.backend.local.port=${pgsqlPort}
+datafeeder.publishing.backend.local.database=${pgsqlDatabase}
+#<schema> is a placeholder to be replaced by the actual schema computed from the "sec-org" request header
+datafeeder.publishing.backend.geoserver.schema=<schema>
+datafeeder.publishing.backend.local.user=${pgsqlUser}
+datafeeder.publishing.backend.local.passwd=${pgsqlPassword}
+datafeeder.publishing.backend.local.preparedStatements=true
+
+# GeoTools DataStore connection params to be used when configuring a GeoServer
+# datastore to access the imported datasets
+datafeeder.publishing.backend.geoserver.dbtype=postgis
+datafeeder.publishing.backend.geoserver.preparedStatements=true
+#<schema> is a placeholder to be replaced by the actual schema computed from the "sec-org" request header
+datafeeder.publishing.backend.geoserver.schema=<schema>
+#datafeeder.publishing.backend.geoserver.jndiReferenceName=java:comp/env/jdbc/datafeeder
+#if a JNDI data source is configured in geoserver, uncomment the above line and comment out the following ones 
+datafeeder.publishing.backend.geoserver.host=${pgsqlHost}
+datafeeder.publishing.backend.geoserver.port=${pgsqlPort}
+datafeeder.publishing.backend.geoserver.database=${pgsqlDatabase}
+datafeeder.publishing.backend.geoserver.user=${pgsqlUser}
+datafeeder.publishing.backend.geoserver.passwd=${pgsqlPassword}
+
+# note how to set a property with spaces: property.prefix.[name\ with\ spaces]=value
+datafeeder.publishing.backend.geoserver.[Loose\ bbox]=false
+datafeeder.publishing.backend.geoserver.[Estimated\ extends]=true
+
+
+#datafeeder.email.send=true #not used yet
+datafeeder.email.ackTemplate=file:${georchestra.datadir}/datafeeder/templates/analysis-started-email-template.txt
+datafeeder.email.analysisFailedTemplate=file:${georchestra.datadir}/datafeeder/templates/analysis-failed-email-template.txt
+datafeeder.email.publishFailedTemplate=file:${georchestra.datadir}/datafeeder/templates/data-publishing-failed-email-template.txt
+datafeeder.email.publishSuccessTemplate=file:${georchestra.datadir}/datafeeder/templates/data-publishing-succeeded-email-template.txt
+
+administratorEmail=georchestra@georchestra.mydomain.org
+
+# Configuration for SMTP email sending of application events
+# Datafeeder will send emails to the user when a job is started, finished, or failed,
+# if the spring.mail.* configuration properties are set.
+
+smtpHost=localhost
```

Comment:
SMTP config should probably be commented out by default, so that it uses global properties instead.
(This comment should be a new issue, I know)

Reply:
commenting out only smtpHost and smtpPort which are both provided by the default.properties, the other ones are not.